### PR TITLE
Ignore COPYING and LICENSE as exact matches or with extensions

### PR DESCRIPTION
**Pull request by carmenbianca**
Related to #881 and https://github.com/fsfe/reuse-docs/issues/139

### CODE REVIEW

**Comment by mxmehl** [comment]
Uh, this will break things. I see many LICENSE.md or LICENSE.txt files around.

I don't quite understand why this change is needed. Do you want to avoid that a file such as `LICENSE-finder.py` is ignored? In this case, we could find a more elaborate regex.

**Comment by carmenbianca** [comment]
> Uh, this will break things. I see many LICENSE.md or LICENSE.txt files around.

Good shout. I changed this so that `COPYING` and `COPYING[.any_ext_here]` are ignored (ditto for `LICENSE`), but `COPYING_foobar` is not ignored.

File: CHANGELOG.md [hunk 1]
```diff
@@ -67,6 +67,10 @@ CLI command and its behaviour. There are no guarantees of stability for the
 
 ### Changed
 
+- Previously, any file that begins with `COPYING` or `LICENSE` was ignored. This
+  has been changed. Now, files like `COPYING_README` are no longer ignored, but
+  `COPYING` and `COPYING.txt` are still ignored (in other words: exact matches,
+  or `COPYING` + a file extension). Idem ditto for `LICENSE`. (#886)
```
Comment (carmenbianca) [inline CHANGELOG.md:73]: This is now in the 3.0.0 change log section…

**Review by mxmehl — APPROVED** [review]
Thanks for fixing this issue :)

**Pull request merged**